2repository/src/integrationTest/resources/docker-compose-alfresco.yml

```diff
@@ -9,7 +9,18 @@ services:
      - postgresql
    environment:
    - SOLR_HOST=solr
+   # this is not actually needed unless custom transformers have been deployed
+   - GLOBAL_legacy.transform.service.enabled=true
```

Comment:
It can act as a partial fallback though for some transformations (transform-misc / TIKA) when the Docker-based transformer is not active / cannot be reached.